qase-jest/src/index.ts

```diff
@@ -69,6 +70,7 @@ class QaseReporter implements Reporter {
         this.options.runComplete = !!this.getEnv(Envs.runComplete) || this.options.runComplete;
         this.api = new QaseApi(
             this.getEnv(Envs.apiToken) || this.options.apiToken || '',
+            this.createHeaders(),
```

Comment:
It’s better to use it at third optional param

Reply:
Done